mailbox/pom.xml

```diff
@@ -595,7 +595,7 @@
             <dependency>
                 <groupId>org.assertj</groupId>
                 <artifactId>assertj-guava</artifactId>
-                <version>3.0.0</version>
+                <version>1.3.1</version>
```

Comment:
I added but Matthieu said that "we put the scope only in dependency section, not in dependencyManagement" so I removed it. What's next?